aya-ebpf-macros/src/lsm_cgroup.rs

```diff
@@ -0,0 +1,87 @@
+use std::borrow::Cow;
+
+use proc_macro2::TokenStream;
+use quote::quote;
+use syn::{ItemFn, Result};
+
+use crate::args::{err_on_unknown_args, pop_string_arg};
+
+pub(crate) struct LsmCgroup {
+    item: ItemFn,
+    hook: Option<String>,
```

Comment:
Aside: I just noticed that the LSM_CGROUP section name format is `lsm_cgroup+` but there is no reference to a footnote next to it. See: https://docs.kernel.org/bpf/libbpf/program_types.html#id14

Kernel selftests indicate it's the same as `lsm` but it would be good to get this corrected in the kernel docs at some point.